### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Psykofloyd**
A french translation for your extension, if you need other language let me know.

### CODE REVIEW

File: language/fr/ajax_shoutbox.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+// Some characters you may want to copy&paste:
+// ’ » “ ” …
+// French translation by Philibert Ollier pollier@student.42.fr
+
+$lang = array_merge(
+	$lang, array(
+		"AJAX_SHOUTBOX"    => "Messagerie instantanée",
+		"SHOUTBOX_MESSAGE" => "Envoyer",
+		"ONLY_AJAX"        => "Desolé, javascript est requis pour utiliser la messagerie instantannée",
+		"AJAX_SHOUTBOX_NO_PERMISSION"   => 'Vous n\'avez pas la permission d\'effectuer l\'action séléctionnée.'
```
Comment (paul999) [inline language/fr/ajax_shoutbox.php]: Please use UTF8 chars instead of '.

File: language/fr/info_acp_ajaxshoutbox.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+// in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
+//
+// Some characters you may want to copy&paste:
+// ’ » “ ” …
+//
+// French translation by Philibert Ollier pollier@student.42.fr
+
+$lang = array_merge(
+	$lang, array(
+		"LOG_AJAX_SHOUTBOX_ERROR"    => "Une erreur s'est produite lors de l'envoi du message au serveur distant."
```
Comment (paul999) [inline language/fr/info_acp_ajaxshoutbox.php]: Same here, please use UTF8

**Comment by paul999** [comment]
Hi, can you also please update your PR with the latest pushed changes?

**Comment by Psykofloyd** [comment]
With pleasure  but i don't know how to insert utf 8 char using sublime text
3 :/

2015-01-17 13:26 GMT+01:00 Paul Sohier <notifications@github.com>:

**Comment by paul999** [comment]
You can copy them from line 37 in the language files.

**Pull request closed without merging**